src/test/regress/pg_regress_main.c

```diff
@@ -108,12 +108,7 @@ static void
 psql_init(void)
 {
 	/* set default regression database name */
-	if (!dblist)
```

Comment:
why is the if check not needed anymore? can't psql_init be called a second time when dblist is not null?